scripts/commit_gitops_manifests.py

```
@@ -0,0 +1,392 @@
+        github_env=_to_path(github_env_raw),
+        dry_run=parse_bool(str(dry_run_raw) if dry_run_raw else None, default=False),
+    )
+
+
+def run_git(args: list[str], cwd: Path, env: dict[str, str] | None = None) -> str:
+    """Run a git command and return stdout.
+
+    Raises RuntimeError on failure.
+    """
+    merged_env = {**os.environ, **(env or {})}
+    result = subprocess.run(
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)